@kiva/kv-activity-feed/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import {
+	connect,
+	StreamClient,
+	GetFeedOptions,
+	NewActivity,
+	StreamFeed,
+	FeedAPIResponse,
+	Activity,
+	APIResponse,
+} from 'getstream';
+
+class ActivityFeedService {
+	private client: StreamClient;
+
+	/**
+	 * Creates an instance of StreamService.
+	 *
+	 * @param {string} apiKey - The GetStream.io API key.
+	 * @param {string} authToken - The auth user token.
+	 * @param {string} appId - The GetStream.io App id.
+	 */
+	constructor(apiKey: string, authToken: string, appId: string) {
+		this.client = connect(apiKey, authToken, appId);
```

Comment:
I'm guessing we want try catch (and log error) around all calls to the 3rd party API